include/power_button.h

```diff
@@ -0,0 +1,12 @@
+#ifndef POWER_BUTTON_H
+#define POWER_BUTTON_H
+
+#include <stdbool.h>
+
+void setup_power_button(void);
+
+bool is_pressed_power_button(void);
+bool is_pressed_power_button_debounced(void);
+bool power_button_state(bool button_state);
+
+#endif // LED_BUTTON_H
```

Comment:
I don't know why you decided to rename everything, when this button doesn't control the power. It just controls the LED, but at least please make sure your fix the comments when renaming.

Reply:
If there is a different name that makes more sense such as led_and_buzzer_button and you are happy with that name then I am open to change. Otherwise, I am pretty sure this particular button handles both the led and buzzer. I also understood that I was stepping out of my boundaries a good amount of times when we had that conversation during the lab. Again, I am open to make changes if you want me to as the code integrator. I hope we are cool with that.